go.mod

```diff
@@ -32,10 +31,9 @@ require (
 	go.uber.org/atomic v1.3.2 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.9.1
-	golang.org/x/exp v0.0.0-20181112044915-a3060d491354
-	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980 // indirect
-	golang.org/x/sync v0.0.0-20181221193216-37e7f081c4d4 // indirect
-	golang.org/x/tools v0.0.0-20181221154417-3ad2d988d5e2 // indirect
+	golang.org/x/exp v0.0.0-20190510132918-efd6b22b2522
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
```

Comment:
I'm not 100% sure why some of these went away, but we should verify that this change is correct before merging.